lib/helpers/focusManager.js

```diff
@@ -30,16 +30,19 @@ function handleFocus () {
 }
 
 export function markForFocusLater () {
-  focusLaterElement = document.activeElement;
+  focusLaterElement.push(document.activeElement);
 }
 
 export function returnFocus () {
+  let toFocus = null;
   try {
-    focusLaterElement.focus();
+    toFocus = focusLaterElement[focusLaterElement.length - 1];
```

Comment:
I wonder if rather than doing this... we could probably consolidate the `.pop()` to here since `pop` returns the last element of the array seems that it'd be the same thing.

Perhaps like this: 
```js
try {
  toFocus = focusLaterElement.pop();
  toFocus.focus();
  return;
}
```